packages/story-editor/src/components/inspector/design/getDesignPanelsForSelection.js

```diff
@@ -85,10 +85,13 @@ function getDesignPanelsForSelection(elements) {
       });
       // In case of default background without media:
     } else {
-      panels.unshift({
-        type: PanelTypes.PAGE_BACKGROUND,
-        Panel: PageBackgroundPanel,
-      });
+      // Avoids adding the pageBackground panel twice.
+      if (!isBackgroundMedia) {
```

Comment:
It checks for `video` and `image` and `gif` as the element type in the `if`s above, doesn't it already mean that it's not background media if it gets to this `else`?

Reply:
Should this be an elseif?

Reply:
@miina It should, and in WordPress it seems to work fine. But when running Karma tests there are a few warnings about `pageBackground` being used two times as the `key` prop, which would indicate that the panel is displayed twice.

Reply:
Hm, does this change fix it? Could you point to an example run with this warning?

Reply:
It does, yes. I don't have a run URL handy for that, but try running the tests yourself with this branch both with and without this particular change here.

Reply:
@swissspidy I was looking into this a bit more and it looks like the issue comes from `formattedTemplatesArray.js` that's used in the tests having incorrect background element values. There are cases where the background element type is `shape`, but `isDefaultBackground` is not set to `true` -- this never happens in the editor. In the editor, the background element is either media, or default background (`isDefaultBackground: true`).

Reply:
Cool, will revert in that case and update that fixture.